modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -87,7 +89,18 @@ def validate
           render json: valid_526_response
         end
 
-        def attachments; end
+        def attachments
+          if params['attachments']&.length&.> 3
+            raise ::Common::Exceptions::UnprocessableEntity.new(detail: 'Too many attachments.')
+          end
+
+          claim = ClaimsApi::AutoEstablishedClaim.get_by_id_or_evss_id(params[:id])
+          raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Resource not found') unless claim
+
+          documents_service(params, claim).process_documents
+
+          render json: claim
```

Comment:
we'll want a [202](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202) here as well since we're processing the upload on background jobs.